tests/migration/eviction_strategy.go

```diff
@@ -361,28 +359,11 @@ var _ = SIGMigrationDescribe("Live Migration", func() {
 					runStressTest(vmi, stressDefaultVMSize, stressDefaultSleepDuration)
 
 					// Mark the control-plane nodes as schedulable so we can migrate there
-					setControlPlaneSchedulability(true)
+					libnode.SetNodeSchedulable(unschedulableControlPlaneNode, virtClient)
```

Comment:
The original idea is that we mark the control-plane node as unschedulable (so that the vmi is not scheduled there).
We don't want the vmi to be scheduled in the control-plane node because we later will drain the node where the vmi is scheduled on.
With cluster with only control plane and one worker node this allow the test to be correctly executed.
Let's see the example:
control-plane node: A
worker node: B
1. Mark A as unschedulable
2. Run the VMI that will be scheduled in B (the only node available)
3. Mark A as schedulable
4. Drain node B
5. VMI can migrate to A (because now it is schedulable)

Without this logic, it can happen that the VMI is scheduled in node A and the drain  of the only one control plane node will have dramatic results xD.

Let's see if we can do domething with the preferredAffinity